plugins/by-name/blink-cmp-copilot/default.nix

```diff
@@ -0,0 +1,32 @@
+{ config, lib, ... }:
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "blink-cmp-copilot";
+  package = "blink-cmp-copilot";
+
+  maintainers = [ lib.maintainers.HeitorAugustoLN ];
+
+  callSetup = false;
+  hasLuaConfig = false;
+  hasSettings = false;
+
+  extraConfig = {
+    warnings =
+      let
+        copilot-lua-cfg = config.plugins.copilot-lua;
+        isEnabled = b: builtins.isBool b && b;
+      in
+      lib.optionals (isEnabled copilot-lua-cfg.suggestion.enabled) [
+        ''
+          It is recommended to disable copilot's `suggestion` module, as it can interfere with
+          completions properly appearing in blink-cmp-copilot.
+        ''
+      ]
+      ++ lib.optionals (isEnabled copilot-lua-cfg.panel.enabled) [
+        ''
+          It is recommended to disable copilot's `panel` module, as it can interfere with completions
+          properly appearing in blink-cmp-copilot.
+        ''
+      ];
+    plugins.copilot-lua.enable = lib.mkDefault true;
```

Comment:
Why put it as `lib.mkDefault true` and not simply `true`? I'd think a user would be warned if they explicitely disable the plugin, I would prefer requiring `lib.mkForce`

Reply:
To be the same behavior as current copilot-cmp: https://github.com/nix-community/nixvim/blob/8938e09db14d510dcc2f266e8b2e738ee527d386/plugins/by-name/copilot-cmp/default.nix#L72